src/Symfony/Component/Form/ChoiceList/Factory/CachingFactoryDecorator.php

```diff
@@ -82,12 +82,12 @@ public function getDecoratedFactory()
     /**
      * {@inheritdoc}
      *
-     * @param callable|Cache\ChoiceValue|null  $value  The callable or static option for
-     *                                                 generating the choice values
-     * @param callable|Cache\ChoiceFilter|null $filter The callable or static option for
-     *                                                 filtering the choices
+     * @param callable|string|Cache\ChoiceValue|null  $value  The callable or static option for
+     *                                                        generating the choice values
+     * @param callable|string|Cache\ChoiceFilter|null $filter The callable or static option for
+     *                                                        filtering the choices
      */
-    public function createListFromChoices(iterable $choices, callable|Cache\ChoiceValue $value = null, callable|Cache\ChoiceFilter $filter = null)
+    public function createListFromChoices(iterable $choices, callable|string|Cache\ChoiceValue $value = null, callable|string|Cache\ChoiceFilter $filter = null)
```

Comment:
Strings are legit for decorators as they convert them to callable, but not for the interface nor the `DefaultChoiceListFactory` expecting only callable.

Reply:
got it, thanks